tests/adapters/test_serial_with_loopback.py

```diff
@@ -0,0 +1,80 @@
+#
+# Test the serial adapter using looped COM ports.
+# This looping needs to be done outside this script. One suggestion is
+# using the application `com0com`.
```

Comment:
Thanks for that test.

Would you mind to expand the setup description a little bit, such that is more easy to replicate it?

Reply:
Sure, you're right. Turned it into a class so the docs description covers it clearly.